docs/topics/settings.txt

```diff
@@ -291,6 +291,23 @@ Note that calling ``django.setup()`` is only necessary if your code is truly
 standalone. When invoked by your Web server, or through :doc:`django-admin
 </ref/django-admin>`, Django will handle this for you.
 
+Note also that you shouldn't call django.setup() in any code that's likely to be
```

Comment:
s/that's likely to be loaded by django itself/that might be imported by Django or other parts of your application/